semeio/jobs/design2params/design2params.py

```diff
@@ -153,14 +153,14 @@ def _read_excel(file_name, sheet_name, header=0):
     """
     try:
         return pd.read_excel(file_name, sheet_name, header=header)
-    except IOError:
-        raise SystemExit("File {} not found".format(file_name))
+    except IOError as err:
+        raise SystemExit("File {} not found".format(file_name)) from err
     except Exception as err:
         raise SystemExit(
             (
                 "File {} is probably not of correct type. Failed with exception '{}'"
             ).format(file_name, str(err))
-        )
+        ) from err
```

Comment:
From the looks of it, the original error message is already included, what do you gain by adding it again?

Reply:
True, perhaps just rewrite it using `from` syntax and drop the `err` in the string.